File: Lib/Telerik/Examples/QuickStart/PropertyGrid/FirstLook/Surface.vb
Imports System.Text
Imports Telerik.WinControls

Namespace Telerik.Examples.WinControls.PropertyGrid.FirstLook
	Public Class Surface
		Inherits RadControl
		Public Element As SurfaceElement

		Protected Overrides Sub CreateChildItems(ByVal parent As RadElement)
			MyBase.CreateChildItems(parent)
			Me.Element = New SurfaceElement()
			parent.Children.Add(Me.Element)
		End Sub
	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/PropertyGrid/FirstLook/Surface.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls;

namespace Telerik.Examples.WinControls.PropertyGrid.FirstLook
{
    public class Surface : RadControl
    {
        public SurfaceElement Element;

        protected override void CreateChildItems(RadElement parent)
        {
            base.CreateChildItems(parent);
            this.Element = new SurfaceElement();
            parent.Children.Add(this.Element);
        }
    }
}
